src/app/cli/interpreter.cpp

```diff
@@ -277,6 +278,7 @@ const std::map<std::string, std::string> &Interpreter::mUsageMap = *new std::map
               "panid conflict <panid>"},
     {"energy", "energy scan <channel-mask> <count> <period> <scan-duration> <dst-addr>\n"
                "energy report [<dst-addr>]"},
+    {"diag", "diag reset [--rloc <rloc>] [--flag <tlvs-flag>]"},
```

Comment:
```suggestion
    {"diag", "diag reset <rloc> <tlvs-flag>"},
```

All the other commands don't use the options like "--rloc" or "--flag". Do we really need those here?

Reply:
The main purpose of adding options is to allow users to omit specifying the order of parameters. In such cases, users could not set the value of option `rloc`, and the leader’s `aloc` will be used by default.